assets/pages/digital-subscription-checkout/digitalSubscriptionCheckout.scss

```diff
@@ -88,6 +83,29 @@
         top: 30%;
       }
     }
+
+    .svg-information {
+      display: inline-block;
+      vertical-align: top;
+      margin-right: 5px;
```

Comment:
can this be `($gu-h-spacing / 2)`?

Reply:
It's actually `($gu-h-spacing / 4)`. I usually don't use `$gu-h-spacing` or `$gu-v-spacing` when the fraction gets lower than half because by that point it's usually something custom rather than a standardised bit of layout.